nipoppy_cli/nipoppy/tabular/doughnut.py

```diff
@@ -145,29 +146,25 @@ def get_bidsified_participants_sessions(
 
 def generate_doughnut(
     manifest: Manifest,
+    dicom_dir_map: DicomDirMap,
     dpath_downloaded: Optional[str | Path] = None,
     dpath_organized: Optional[str | Path] = None,
     dpath_bidsified: Optional[str | Path] = None,
```

Comment:
Do these need to change to support python3.9?

Reply:
They will be once I get the latest changes from `main`!